src/build/tasks/copy.ts

```diff
@@ -1,11 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import { execSync } from 'child_process';
-import { existsSync } from 'fs';
+import { existsSync, mkdirSync } from 'fs';
 
 export function copyAllFiles(from: string, to: string) {
   if (!existsSync(to)) {
-    execSync(`mkdir -p ${to}`, { stdio: 'inherit' });
+    mkdirSync(to, { recursive: true });
   }
   execSync(`cp -r ${from}/. ${to}`, { stdio: 'inherit' });
```

Comment:
Hi @jasmaa,
thanks for the PR. 
Is `cp` also not compatible with windows?

Reply:
Looks like I have cygwin on my machine so cp was working. Added a change to use fs.cpSync